.gitignore

```diff
@@ -4,7 +4,6 @@ pom.xml.versionsBackup
 TESTS-TestSuites.xml
 *.db
 */dat/out
-/.settings/
 .idea
```

Comment:
Settings exclusion has been removed, which does not match with the current version on GitLab. Also, the settings folders are in the remote. Are you going to change the approach to solve the problem that motivated the removal of settings in the remote?

Reply:
I removed the exclusion because I saw the folder .settings in other projects of the giis-uniovi repository, so I wasn't sure of the correct approach. Would it be prefferable to remove it as in Gitlab and regain the exclusion?

Reply:
Both options are possible. If there are many developers with different environments, it's better to remove these deve env settings. In our case, as we use Eclipse, is also ok to keep the settings. But the problem arise with modevo. Remember that the problem was that opening from eclipse WITHOUT the atl plugins, caused problems, your solution was to remove the settings, while keeping .classpath and project. It is up to you the decision, I'm only remarking that is a different configuration than the one you have in the GitLab project that is being migrated.

Reply:
Ok, I think the best approach is to keep as similar as possible to the Gitlab version, so I removed them and include the exclusion again.